tests/client/test_workspaces.py

```diff
@@ -122,3 +122,16 @@ def test_workspace_delete_user(owner: "ServerUser") -> None:
 
     with pytest.raises(ValueError, match="Either the user with id="):
         workspace.delete_user(owner.id)
+
+
+def test_print_workspace(owner: "ServerUser"):
+    workspace = WorkspaceFactory.create(name="test_workspace")
+
+    import argilla as rg
+
+    rg.init(api_key=owner.api_key)
```

Comment:
```suggestion
    ArgillaSingleton.init(api_key=owner.api_key)
```

Reply:
Not sure about this change. If we're coding a test for the client, we should use the way we're using normally the client, so `import argilla as rg; rg.....`

Reply:
I guess we should try to avoid to specifically `import argilla as rg` inside our code base, as that's just for the users and examples/tutorials, internally we should make everything more explicit to have some traceability IMO